algorithms/closest_pair.py

```diff
@@ -84,7 +84,12 @@ def _distance_between(point_1, point_2):
 
```

Comment:
https://github.com/Wynndow/algorithms/pull/2/files#diff-01016d6362c2e04e50a730472494b7c2R58

`range(1, min(7, len(potential_closest_split_pairs)-i))` is quite confusing.

Rather than referring to the pair
`potential_closest_split_pairs[i]` and `potential_closest_split_pairs[i + j]`,
you could just change what `j` is and use `potential_closest_split_pairs[i]` and `potential_closest_split_pairs[j]`.

I think the range should go from `i` to `len(potential_closest_split_pairs)`, like in the brute force function.

What's the 7 for?